pydevd_file_utils.py

```diff
@@ -963,3 +964,9 @@ def get_package_dir(mod_name):
         if os.path.isdir(mod_path):
             return mod_path
     return None
+
+def is_pydevd_path(path):
```

Comment:
This was necessary for the `<string>` module fix. We only want to skip `<string>` modules that come from pydevd